python-main.js

```diff
@@ -1434,16 +1439,32 @@ function web_editor(config) {
         $("#command-share").click(function () {
             doShare();
         });
-        $("#command-connect").click(function () {
-            if ($("#command-connect").length) {
-                doConnect();
-            } else {
-                doDisconnect();
+        if (navigator.usb) {
+            $("#command-connect").click(function () {
+                if ($("#command-connect").length) {
+                    doConnect();
+                } else {
+                    doDisconnect();
+                }
+            }); 
+            $("#command-serial").click(function () {
+                doSerial();
+            });  
+
+        }else{
+            function buttonClicked(){
+                openModalMsg("WebUSB not supported", "With WebUSB you can program your micro:bit and connect to the serial console directly from the online editor.<br/>However this feature is not supported on this browser.", "help.html#WebUSB");
```

Comment:
Let's change the last sentence to "Unfortunately, WebUSB is not supported in this browser. We recommend Chrome, or a Chrome-based browser to use WebUSB."

Reply:
And let's keep the title simple to just "WebUSB".